src/shared/ofi_str.c

```diff
@@ -199,3 +200,50 @@ void ofi_free_string_array(char **s)
 	/* and then the actual array of pointers */
 	free(s);
 }
+
+char *ofi_tostr_size(char *str, size_t len, uint64_t size)
+{
+	uint64_t base = 0;
+	uint64_t fraction = 0;
+	char mag;
+
+	memset(str, '\0', len);
```

Comment:
We don't need to memset the entire character array, which may be large.  Just ensure that the array is null terminated before exiting.